File: LyubomirMihaylov_1601261068/Bubble_Sort/bubble_sort.py

arr = [ 13, 44, 5, 7, 67, 2, 34, 545, 5555, 2, 8, 89, 5, 1 ]

def bubbleSort(arr):
    n = len(arr)
    for i in range(n):
        for j in range(0, n-1):
            if arr[j] > arr[j+1] :
                arr[j], arr[j+1] = arr[j+1], arr[j]
    print(arr)
				
bubbleSort(arr)

File: IvanBonchev1601261044/BubbleSort/BubbleSort/BubbleSort.cs
using System;

namespace ConsoleApp1
{
    class Program
    {
        static void Main(string[] args)
        {
            int[] arr = new int[] { 7, 10, 3, 9, 1, 888, 29, 0 };

            BubbleSort(arr);

            for (int i = 0; i < arr.Length; i++)
            {
                Console.Write(arr[i] + " ");
            }
            Console.ReadLine();
        }

        static void BubbleSort(int[] array)
        {
            for (int i = 0; i < array.Length - 1; i++)
            {
                for (int j = i + 1; j < array.Length - 2; j++)
                {

                    if (array[i] > array[j])
                    {
                        int temp = array[i];
                        array[i] = array[j];
                        array[j] = temp;
                    }
                }

            }

        }

    }
}
